src/main/java/org/opentripplanner/gtfs/RepairStopTimesForEachTripOperation.java

```diff
@@ -21,6 +21,7 @@
 import org.opentripplanner.transit.model.site.Stop;
```

Comment:
Can we take this opportunity to rename this class? It's not reparing anything anymore.